/home/ubuntu/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/matplotlib/__init__.py:1067: UserWarning: Duplicate key in file "/home/ubuntu/.config/matplotlib/matplotlibrc", line #2
  (fname, cnt))
/home/ubuntu/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/matplotlib/__init__.py:1067: UserWarning: Duplicate key in file "/home/ubuntu/.config/matplotlib/matplotlibrc", line #3
  (fname, cnt))
epoch 0: loss 0.029863520973603987
epoch 1: loss 0.029652385329356684
epoch 2: loss 0.02951079372146566
epoch 3: loss 0.029329293185360893
epoch 4: loss 0.029110800858684456
epoch 5: loss 0.02892211449201203
epoch 6: loss 0.028679401819187586
epoch 7: loss 0.028443110270527802
epoch 8: loss 0.028189080777638244
epoch 9: loss 0.027935671979637807
epoch 10: loss 0.027722335816601978
epoch 11: loss 0.02747626660206347
epoch 12: loss 0.02727135727455231
epoch 13: loss 0.027031146168787455
epoch 14: loss 0.02685142676625685
epoch 15: loss 0.02667496647474633
epoch 16: loss 0.026504738892715547
epoch 17: loss 0.026343605849281346
epoch 18: loss 0.02616550144608169
epoch 19: loss 0.026008075462152166
